src/srf/boolean.cpp

```diff
@@ -63,11 +66,24 @@ SCurve SCurve::MakeCopySplitAgainst(SShell *agnstA, SShell *agnstB,
 
     // First find any vertex that lies on our curve.
     List<SInter> vertpts = {};
-    if(agnstA)
-        FindVertsOnCurve(&vertpts, this, agnstA);
-    if(agnstB)
-        FindVertsOnCurve(&vertpts, this, agnstB);
-
+    if(isExact) {
+        // get a bounding box for our curve
+        Vector cmax = Vector::From(-1e10, -1e10, -1e10);
```

Comment:
There are defines for "very big" and "very small" don't use literals in the code apart from 0 and 1